packages/features/tasker/tasks/executeAIPhoneCall.ts

```diff
@@ -147,83 +185,64 @@ export async function executeAIPhoneCall(payload: string) {
     }
 
     // TODO: add better rate limiting for AI phone calls
-    if (rateLimitIdentifier) {
-      await checkRateLimitAndThrowError({
-        rateLimitingType: "core",
-        identifier: rateLimitIdentifier,
-      });
-    }
+    await checkRateLimitAndThrowError({
+      rateLimitingType: "core",
+      identifier: rateLimitIdentifier,
+    });
+
+    // form triggers don't have a booking
+    const booking = workflowReminder.booking as BookingWithRelations | null;
+    const responses = data.responses;
```

Comment:
we need a better name of this variable as this can only be `FORM_SUBMITTED_WEBHOOK_RESPONSES`